src/persistedState/codecs.ts

```diff
@@ -107,6 +107,18 @@ export const persistedStateCodecNew = buildTypeCodec({
 			selectedCaseHash: null,
 		},
 	),
+	panelView: withFallback(
+		buildTypeCodec({
```

Comment:
this should be a union codec representing the following states:
* a meaningful codemod description to render
* a meaningful job to render
* nothing to render (meaning the panel will never be displayed); default value for `panel`

with such a union our need to deal with nullability is drastically reduced

Reply:
you could probably store the `codemodHash` instead of `docs` to point to the codemod you wanna display. The selector should take care of extracting the information.